src/workerd/api/streams/internal.c++

```diff
@@ -1116,9 +1114,8 @@ jsg::Promise<void> WritableStreamInternalController::flush(jsg::Lock& js, bool m
       if (markAsHandled) {
         prp.promise.markAsHandled(js);
       }
-      queue.push_back(
-          WriteEvent{.outputLock = IoContext::current().waitForOutputLocksIfNecessaryIoOwn(),
-            .event = Flush{.promise = kj::mv(prp.resolver)}});
+      queue.push(WriteEvent{.outputLock = IoContext::current().waitForOutputLocksIfNecessaryIoOwn(),
+        .event = Flush{.promise = kj::mv(prp.resolver)}});
```

Comment:
Nit: clang-format might produce less ugly results if you add trailing commas after the field initializers

Reply:
Yeah, next time I'm poking around in this code I'll see about tweaking that.